app/assets/javascripts/teaspoon-filterer.js

```diff
@@ -0,0 +1,55 @@
+(function() {
+  var Teaspoon,
+    __bind = function(fn, me){ return function(){ return fn.apply(me, arguments); }; };
+
+  Teaspoon = {};
+
+  Teaspoon.Filterer = (function() {
+    function Filterer() {
+      this.filter = __bind(this.filter, this);
+      this.files = document.querySelectorAll("#teaspoon-suite-list .file a");
+      this.input = document.querySelector("#teaspoon-filter-input");
+      this.input.value = "";
+      this.input.onkeyup = this.filter;
+    }
+
+    Filterer.prototype.filter = function() {
+      var file, _i, _len, _ref, _results;
+      _ref = this.files;
+      _results = [];
+      for (_i = 0, _len = _ref.length; _i < _len; _i++) {
+        file = _ref[_i];
+        if (LiquidMetal.score(file.innerHTML, this.input.value) > 0) {
+          _results.push(file.parentNode.style.display = "block");
+        } else {
+          _results.push(file.parentNode.style.display = "none");
+        }
+      }
+      return _results;
+    };
+
+    return Filterer;
+
+  })();
+
+  window.onload = function() {
+    return new Teaspoon.Filterer();
+  };
+
+
+  /*!
+   * LiquidMetal
+   * Copyright (c) 2009, Ryan McGeary (ryanonjavascript -[at]- mcgeary [*dot*] org)
+   */
+  var LiquidMetal=function(){var l=0.0;var m=1.0;var n=0.8;var o=0.9;var p=0.85;return{score:function(a,b){if(b.length==0)
+    return n;if(b.length>a.length)return l;var c=this.buildScoreArray(a,b);var d=0.0;for(var i=0;i<c.length;i++){d+=c[i]}
```

Comment:
Line is too long.<br>Expected '{' and instead saw 'return'.<br>Missing semicolon.